### PR TITLE
GEOPY-415: Fill in default ui.json tooltips for inversion parameters

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-415 - Fill in default ui.json tooltips for inversion parameters**

### CODE REVIEW

File: simpeg_drivers/constants.py [hunk 1]
```diff
@@ -144,6 +136,7 @@
         "value": "",
         "optional": True,
         "enabled": True,
+        "tooltip": "Select a topography object to define the active cells for inversion.  The user may also provide active cells directly by disabling the parameter.  If no elevation channel is selected, the topography will be set from the geometry of the selected object.",
```
Comment (domfournier) [inline simpeg_drivers/constants.py:139]: Not sure a comment this long will render well. Just focus on the parameter itself and not its children.

File: simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json [hunk 2]
```diff
@@ -324,7 +344,8 @@
         "max": 2.0,
         "precision": 2,
         "lineEdit": false,
-        "enabled": true
+        "enabled": true,
+        "tooltip": "Lp-norm used in the objective function.  0.0 is equivalent to the L0-norm, and 2.0 is equivalent to the L2-norm, but any values in between are also valid.  May be provided as either a float or a model living on th cells of the selected mesh."
```
Comment (domfournier) [inline simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json:348]: Also too long

File: simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json [hunk 3]
```diff
@@ -334,7 +355,8 @@
         "group": "Sparse/blocky model",
         "label": "Gradient type",
         "value": "total",
-        "verbose": 3
+        "verbose": 3,
+        "tooltip": "This property specifies whether the IRLS weights for sparse smoothness regularization(s) terms are updated using the total gradient (*'total'*) or using the partial gradients along their smoothing orientations (*'components'*)."
```
Comment (domfournier) [inline simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json:359]: Same

**Review by domfournier — COMMENTED** [review]
I truly appreciate the work, but tooltips can't be too long. Some of this stuff should just be transferred to the docs.

File: simpeg_drivers/constants.py (hunk 1 above)
Comment (benk-mira) [reply simpeg_drivers/constants.py:139]: well, we could format the tooltip window better..  but sure - I'll shorten

File: simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json (hunk 3 above)
Comment (benk-mira) [reply simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json]: updated

File: simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json (hunk 2 above)
Comment (benk-mira) [reply simpeg_drivers-assets/uijson/direct_current_inversion_2d.ui.json]: updated

File: simpeg_drivers/constants.py (hunk 1 above)
Comment (benk-mira) [reply simpeg_drivers/constants.py]: done everywhere

**Review by domfournier — APPROVED** [review]

**Pull request merged**